table/block_based/block_based_table_builder.cc

```diff
@@ -2084,6 +2083,12 @@ const char* BlockBasedTableBuilder::GetFileChecksumFuncName() const {
     return kUnknownFileChecksumFuncName;
   }
 }
+void BlockBasedTableBuilder::SetSeqnoTimeTableProperties(
+    const std::string& encoded_seqno_to_time_mapping,
+    uint64_t oldest_ancestor_time) {
+  rep_->props.seqno_to_time_mapping = encoded_seqno_to_time_mapping;
+  rep_->props.creation_time = oldest_ancestor_time;
```

Comment:
I think PlainTable should be able to support it too. It will be useful at least for the deletion ttl work. Maybe add a comment somewhere for a TODO work.

Reply:
Added a TODO, I think it should not be difficult to add, just need more tests for that.